k2/csrc/array_ops_benchmark.cu

```diff
@@ -0,0 +1,36 @@
+/**
+ * @brief Benchmarks for k2 APIs.
```

Comment:
not sure if we'd like put those files under `csrc`, maybe a folder like `benchmark`. I just feel compared to `**_test.cu`, those benchmark files would not be read/used by normal users.

Reply:
I am fine with where to put these files.